draft-ietf-quic-tls.md

```diff
@@ -1538,27 +1538,34 @@ discarded.
 
 ## Minimum Key Update Frequency
 
-Key updates MUST be initiated before usage limits on packet protection keys are
-exceeded. For the cipher suites mentioned in this document, the limits in
-Section 5.5 of {{!TLS13}} apply. {{!TLS13}} does not specify a limit for
-AEAD_AES_128_CCM, but the analysis in {{ccm-bounds}} shows that a limit of 2^23
-packets can be used to obtain the same confidentiality protection as the limits
-specified in TLS.
```

Comment:
I didn't get enough time to do this today, but what I think we need here is some of the framing from this.  And that likely comes with a new section title and maybe even some restructuring.

Reply:
I reorganized things a bit. There's a parallel flow for the connection and integrity limit text now that think works. @ekr, what do you think of the new text?